optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -293,6 +333,47 @@ export const TrialTable: FC<{
             />
           </Box>
         </Grid>
+        {trials[index].state === ("Running" as TrialState) ? (
+          <Grid item xs={12}>
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Trial tell operations
+              </Typography>
+              <form onSubmit={handleSubmit}>
+                <Box margin={1}>
+                  <Stack direction="row" spacing={1}>
+                    {objectiveValues.map((value, i) => (
+                      <TextField
+                        id={`objective-${i}`}
+                        key={`objective-${i}`}
+                        label={`Objective ${i}`}
+                        type="number"
```

Comment:
Floating point numbers seems to be prohibited when using `type="number"`. You need to set `step` or `pattern` attribute as well.
<img width="605" alt="Screenshot 2023-01-16 14 30 54" src="https://user-images.githubusercontent.com/5564044/212605992-8ba37182-4a9a-4d0f-8715-0959593c3545.png">

Reply:
I used the following one and it was fixed.

```
    type="text"
    pattern="^([1-9]\d*|0)(\.\d+)?$"
```